notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -1760,6 +1767,18 @@
     "        uvps.append(uvp)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "a3d1811e",
+   "metadata": {},
+   "source": [
+    "### A note on estimating $P_{SN}$:\n",
+    "\n",
+    "[Tan et al. (2021)](https://arxiv.org/abs/2103.09941) showed that $P_{NS}^2 = \\sqrt{2}P_S P_N + P_N^2$ (Eq. 30). The challenge is estimating $P_S$. Any power spectrum measurement has both noise and signal in it, so even with the imposition of the prior that $P_S$ is real and non-negative, we expect double counting of the noise. [Tan et al. (2021)](https://arxiv.org/abs/2103.09941) shows that the expectation value of the 1st term on the LHS of Eq. 30 has an expectation value of $P_N^2 / \\sqrt{\\pi}$ for Gaussian-distributed power spectra (see Eq. 31) and $P_N^2 / 2$ for Laplacian-distributed power spectra. The former is appropriate after sufficient time averaging, the latter for single integrations.\n",
```

Comment:
$P_{NS}$ in $P_{NS}^2 = \\sqrt{2}P_S P_N + P_N^2$ should be $P_{SN}$

Reply:
Fixed. Good catch.